app/controllers/departments_controller.rb

```diff
@@ -2,63 +2,46 @@ class DepartmentsController < ApplicationController
   before_action :set_department, only: [:show, :edit, :update, :destroy]
   before_action :admin_employee
 
-  # GET /departments
-  # GET /departments.json
   def index
     @departments = Department.all
   end
 
-  # GET /departments/1
-  # GET /departments/1.json
   def show
   end
 
-  # GET /departments/new
   def new
     @department = Department.new
   end
 
-  # GET /departments/1/edit
   def edit
   end
 
-  # POST /departments
-  # POST /departments.json
   def create
     @department = Department.new(department_params)
 
     respond_to do |format|
```

Comment:
If you aren't using multiple formats of responses, it may be cleaner just to leave this out entirely.